changelogs/fragments/714-infinity.yaml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - postgres - add support for postgres ``infinity`` timestamps by replacing them with ``datetime.min`` / ``datetime.max`` values (https://github.com/ansible-collections/community.postgresql/pull/714)
```

```suggestion
  - postgres - add support for postgres ``infinity`` timestamps by replacing them with ``datetime.min`` / ``datetime.max`` values (https://github.com/ansible-collections/community.postgresql/pull/714).
```
for changelog consistency